system-parachains/asset-hubs/asset-hub-polkadot/src/bridge_to_ethereum_config.rs

```diff
@@ -0,0 +1,208 @@
+use xcm_executor::XcmExecutor;
+
+#[cfg(feature = "runtime-benchmarks")]
+pub mod benchmark_helpers {
+	use crate::{
+		xcm_config::LocationToAccountId, AccountId, AssetConversion, Balances, ForeignAssets,
+		RuntimeOrigin,
+	};
+	use alloc::boxed::Box;
+	use codec::Encode;
+	use xcm::prelude::*;
+	use xcm_executor::traits::ConvertLocation;
+
+	pub struct DoNothingRouter;
+	impl SendXcm for DoNothingRouter {
+		type Ticket = Xcm<()>;
+
+		fn validate(
+			_dest: &mut Option<Location>,
+			xcm: &mut Option<Xcm<()>>,
+		) -> SendResult<Self::Ticket> {
+			Ok((xcm.clone().unwrap(), Assets::new()))
+		}
+		fn deliver(xcm: Xcm<()>) -> Result<XcmHash, SendError> {
+			let hash = xcm.using_encoded(sp_io::hashing::blake2_256);
+			Ok(hash)
+		}
+	}
+
+	impl snowbridge_pallet_system_frontend::BenchmarkHelper<RuntimeOrigin, AccountId> for () {
+		fn make_xcm_origin(location: Location) -> RuntimeOrigin {
+			RuntimeOrigin::from(pallet_xcm::Origin::Xcm(location))
+		}
+
+		fn initialize_storage(asset_location: Location, asset_owner: Location) {
+			let asset_owner = LocationToAccountId::convert_location(&asset_owner).unwrap();
+			ForeignAssets::force_create(
+				RuntimeOrigin::root(),
+				asset_location,
+				asset_owner.into(),
+				true,
+				1,
+			)
+			.unwrap()
+		}
+
+		fn setup_pools(caller: AccountId, asset: Location) {
+			// Prefund the caller's account with DOT
+			Balances::force_set_balance(
+				RuntimeOrigin::root(),
+				caller.clone().into(),
+				10_000_000_000_000,
+			)
+			.unwrap();
+
+			let asset_owner = caller.clone();
+			ForeignAssets::force_create(
+				RuntimeOrigin::root(),
+				asset.clone(),
+				asset_owner.clone().into(),
+				true,
+				1,
+			)
+			.unwrap();
+
+			let signed_owner = RuntimeOrigin::signed(asset_owner.clone());
+
+			// Prefund the asset owner's account with DOT and Ether to create the pools
+			ForeignAssets::mint(
+				signed_owner.clone(),
+				asset.clone(),
+				asset_owner.clone().into(),
+				10_000_000_000_000,
+			)
+			.unwrap();
+			Balances::force_set_balance(
+				RuntimeOrigin::root(),
+				asset_owner.clone().into(),
+				10_000_000_000_000,
+			)
+			.unwrap();
+
+			// Create the pool so the swap will succeed
+			let native_asset: Location = Parent.into();
+			AssetConversion::create_pool(
+				signed_owner.clone(),
+				Box::new(native_asset.clone()),
+				Box::new(asset.clone()),
+			)
+			.unwrap();
+			AssetConversion::add_liquidity(
+				signed_owner,
+				Box::new(native_asset),
+				Box::new(asset),
+				1_000_000_000_000,
+				2_000_000_000_000,
+				0,
+				0,
+				asset_owner,
+			)
+			.unwrap();
+		}
+	}
+}
+
+parameter_types! {
+	/// Fee asset for the execution cost on ethereum
+	pub storage FeeAsset: Location = Location::new(
+			2,
+			[
+				EthereumNetwork::get().into(),
+			],
+	);
+	pub BridgeHubLocation: Location = Location::new(1, [Parachain(polkadot_runtime_constants::system_parachain::BRIDGE_HUB_ID)]);
```

Comment:
@claravanstaden use `xcm_config::bridging::SiblingBridgeHub` instead please
```suggestion
```

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/796/commits/f92ff86e79826d83f1f7bb379e9a8bfc30362355.